README.md

```diff
@@ -21,14 +21,16 @@ See [issue #1](https://github.com/paulodiovani/hacktoberrank/issues/1) for more
 npm install
 ```
 
+Install Redis: [Redis Documentation](https://redis.io/topics/quickstart)
```

Comment:
Instead of _install_ instructions, add a _Dependency_ section:

## Dependencies

- [Node.js](https://nodejs.org) >= 10.16.0
- [Redis](https://redis.io)

Reply:
FYI, if you need redis for testing, you can easily start a server with docker:

```bash
docker run --rm -d -p 6379:6379 --name redis-server redis:alpine

Reply:
done! 

Thanks 👍